src/Service/ServiceListenerFactory.php

```diff
@@ -94,8 +94,6 @@ class ServiceListenerFactory implements FactoryInterface
             'Zend\View\Resolver\TemplatePathStack'       => 'ViewTemplatePathStack',
             'Zend\View\Resolver\AggregateResolver'       => 'ViewResolver',
             'Zend\View\Resolver\ResolverInterface'       => 'ViewResolver',
-            'ControllerManager'                          => 'ControllerLoader',
-        ],
```

Comment:
yes, at ZF3, the old one ( ControllerLoader ) will no longer used. https://github.com/zendframework/zf2/pull/4962